infra/helm/meshdb/example.values.yaml

```diff
@@ -0,0 +1,119 @@
+# Default values for meshdb.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+meshdb_app_namespace: "meshdbdev0"
+
+pg:
+  dbname: meshdb
+  user: meshdb
+  port: "5432"
+  pvc_name: "meshdb-postgres-pvc"
+  pvc_size: "20Gi"
+  liveness_probe: "true"
+  podSecurityContext: {}
+  # fsGroup: 2000
+  securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+  resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
```

Comment:
Delete comment? Maybe we should talk about this. Our resources are going to be so dummy small. Limiting to like 500mcore and 500mb of RAM is probably fine.

Reply:
Entire file gone